src/app.js

```diff
@@ -1 +1,32 @@
 'use strict';
+
+(() => {
+  const params = process.argv.slice(2);
+
+  let result = 0;
+
+  if (params[0] === 'add') {
+    for (let i = 1; i < params.length; i++) {
+      if (isNaN(+params[i])) {
+        break;
```

Comment:
> Skip all not numeric value

I guess, we should continue our loop, if we get not numeric number
```suggestion
        continue;
```

Reply:
yes...my bad =(